pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -2467,6 +2492,13 @@ func (gce *GCECloud) getInstancesByNames(names []string) ([]*gceInstance, error)
 			pageToken = res.NextPageToken
 			for _, i := range res.Items {
 				name := i.Name
+				if inst, ok := instances[name]; !ok || inst != nil {
+					if inst != nil {
+						glog.Warningf("GCE returned duplicate result for instance %q", name)
```

Comment:
I went ahead and took it out. If the other case ever comes up, hopefully it'll be obvious (or non-harmful, e.g. duplicate results but all the results).